src/sentry/static/sentry/less/includes/bootstrap/variables.less

```diff
@@ -40,12 +40,9 @@
 //== Typography
 //
 //## Font, line-height, and color for body text, headings, and more.
-
-@font-family-sans-serif: 'Helvetica Neue', Helvetica, Arial, sans-serif;
-@font-family-serif: Georgia, 'Times New Roman', Times, serif;
 //** Default monospace fonts for `<code>`, `<kbd>`, and `<pre>`.
 @font-family-monospace: 'IBM Plex', Menlo, Monaco, Consolas, 'Courier New', monospace;
-@font-family-base: @font-family-sans-serif;
+@font-family-base: 'Rubik', 'Avenir Next', 'Helvetica Neue', sans-serif;
```

Comment:
Going to remove these variables eventually, but we need to delete each part of Bootstrap before I want to do that.